helix-view/src/document.rs

```diff
@@ -1394,6 +1446,8 @@ impl Document {
         };
 
         self.inlay_hints_oudated = true;
+        self.color_swatches_outdated = true;
```

Comment:
I see now this is were you set this var. but i am still not sure if we do actually need it or if we cannot just use IDs